src/style.css

```diff
@@ -15,30 +23,47 @@ div#root {
     font-size: 1.375em;
     color: rgba(0,0,0,.74);
     background-image: none;
-    background-color: transparent;
+    background-color: rgba(255,255,255,.96);
     border: none;
-    border-right: 1px solid rgba(0,0,0,.15);
-    border-bottom: 1px solid rgba(0,0,0,.15);
+    border-right: 2px solid rgba(0,0,0,.06);
+    border-bottom: 2px solid rgba(0,0,0,.06);
     box-sizing: border-box;
-    transition-duration: 355ms;
+    
 }
 
+.btn-0.size-2 {
+    min-width: 10em;
+    min-height: 5em;
+}
+.btn-0.last {
+    background-color: orange;
+}
 .display {
     font-size: 3em;
     min-height: 2.5em;
     display: flex;
     flex-direction: column;
     justify-content: center;
-    padding-left: 1em;
-    background-color: rgba(0,0,0,.06);
-    color: rgba(0,0,0,.34);
+    padding-left: .56em;
+    background-color: rgba(255,255,255,.14);
+    color: rgba(255,255,255,.34);
+    border: 1px solid rgba(0,0,0,.15);
+    box-sizing: border-box;
+    margin-left: 0.25px;
+    font-weight: 500;
+
 }
 
 button:focus, input[type="button"]:focus {
     outline: none;
 }
 button:focus, input[type="button"]:active {
-    transition-duration: 200ms;
-    color:yellow;
-    background-color: #F0F0F5;
+  
+    color:#F0F0F5;
+    
+
+}
+
+.buttonpanel {
```

Comment:
This should use flexbox display ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)

The row height should be 100px or it's equivalent ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/rowHeight.png)